src/DefaultRESTClient.ts

```diff
@@ -182,6 +182,19 @@ export default class DefaultRESTClient<RT, ERT> implements IRESTClient<RT, ERT>
     return this;
   }
 
+  /**
```

Comment:
our other clients tolerated null/undefined values OOB here, our TS one did not. To minimize blast radius, added a separate method